tests/optimizers/test_layer_adaptive_scaling.py

```diff
@@ -0,0 +1,45 @@
+import pytest
```

Comment:
@nateraw @williamFalcon I don't think this file is required because:

1) we are changing the lars implementation to the official apex version
2) optimizers should be unit tested for convergence and not just params

Reply:
I was getting yelled at for coverage and these 4 Exceptions were marked as uncovered. thats the only reason I did this. If this gets merged before you start using the other implementation, then you can delete this if ya want. For now, lets just leave this so I keep my checks passing haha. 

If you see anything glaring in my branch that would be easy to add tests for that would increase my coverage enough for it to pass, I'm all ears. I just refuse to have failing checks if I can help it.